spec/flipper/adapters/active_support_cache_store_spec.rb

```diff
@@ -9,7 +9,7 @@
   let(:memory_adapter) do
     Flipper::Adapters::OperationLogger.new(Flipper::Adapters::Memory.new)
   end
-  let(:cache) { ActiveSupport::Cache::DalliStore.new(ENV['MEMCACHED_URL']) }
+  let(:cache) { ActiveSupport::Cache::MemoryStore.new }
```

Comment:
DalliStore is deprecated (https://github.com/petergoldstein/dalli/issues/573) in favor of MemCacheStore, but I swapped it with `MemoryStore` instead to avoid the dependency on memcached.

Reply:
👍